### ISSUE TITLE
Is an optional field for 'down-payment' a desired feature?

### ISSUE THREAD

**Issue by mishasaggi** [jpederson/Accrue.js#23]

**Comment by troymcginnis** [comment]
Yeah, for sure. I was just starting to use Accrue and thought about how nice it would be to have this...

**Comment by mishasaggi** [comment]
Great! I submitted a pull request to the original repo sometime ago, it hasn't been reviewed yet. but I can submit to your fork tonight so you can start using it. You can make a separate branch for it if you like. Let me know the name.

**Comment by troymcginnis** [comment]
@mishasaggi That would be awesome. I'm forked over here at: https://github.com/troymcginnis/Accrue.js. Thanks so much!

**Comment by mishasaggi** [comment]
@troymcginnis submitted a pull request to your fork.
